internal/pkg/cli/init.go

```diff
@@ -229,6 +229,12 @@ func BuildInitCmd() *cobra.Command {
 		Use:   "init",
 		Short: "Create a new ECS application.",
 		PreRunE: runCmdE(func(cmd *cobra.Command, args []string) error {
+			if *opts.dockerfilePath == "" {
+				_, err = validateDockerfiles(&afero.Afero{Fs: afero.NewOsFs()})
+				if err != nil {
+					fmt.Errorf("fail to find Dockerfiles: %w", err)
```

Comment:
Similar to previous comment, we're already wrapping error conditions in the `validateDockerfiles` function, so I don't think wrapping again is providing value.